x-pack/plugins/monitoring/public/components/cluster/listing/listing.js

```diff
@@ -21,16 +21,31 @@ import {
   EuiIcon,
   EuiToolTip,
 } from '@elastic/eui';
-import { EuiMonitoringTable } from '../../table';
+
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
+
+import { Legacy } from '../../../legacy_shims';
+import { EuiMonitoringTable } from '../../table';
 import { AlertsStatus } from '../../../alerts/status';
 import { STANDALONE_CLUSTER_CLUSTER_UUID } from '../../../../common/constants';
 import { getSafeForExternalLink } from '../../../lib/get_safe_for_external_link';
-import './listing.scss';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
 import { toMountPoint } from '@kbn/react-kibana-mount';
 
+const clusterCellExpired = (theme) => css`
+  color: ${theme.euiTheme.colors.textParagraph};
+`;
+
+const clusterCellLicense = (theme) => css`
+  // euiFontSize
```

Comment:
TODO: check all cases of Sass `euiFontSize()` and replace with appropriate alternative.